facades/PC/build.gradle.kts

```diff
@@ -129,109 +123,45 @@ tasks.register<JavaExec>("game") {
     main = mainClassName
     workingDir = rootDir
 
+    classpath(sourceSets["main"].runtimeClasspath)
```

Comment:
so that's ... all .. we need?

Reply:
Yeah.
Facade classes + dependencies.

Reply:
and I expect this is the default classpath of the `run` task.

What I don't know is how to take advantage of those defaults for making and customizing several different tasks. Last time I looked, `run` wasn't its own Class of task, it was a customized JavaExec a lot like this.

Either way, I think this is a big improvement! We can update it later if we figure out how bring it closer to defaults, and we won't have to worry about finding all the duplicates if/when we do.

Reply:
Yeah. This is default parameter in unknown place.


JavaExec have default as null there.
(It can be used for run external java code)